server/appartners_server.py

```diff
@@ -95,5 +118,70 @@ def login():
         return jsonify({"errorMessage": str(e)}), 500
 
 
+@app.route('/create-profile', methods=['POST', 'OPTIONS'])
+def create_profile():
+    try:

```

Comment:
would change it to update_profile()
when a person signup we need to create an empty profile for him.
after it when he update the profile page, we need to update the info.

Reply:
change post to put